### PR TITLE
Good code branch

### PR DESCRIPTION
**Pull request by MarcusTita**
This is the best code ver

### CODE REVIEW

File: firstFile.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 //here is my first comment
 
 //another comment
-
+const log = 'loggers';
```
Comment (AlfredMadere) [inline firstFile.js:4]: I hate this variable name

Comment (MarcusTita) [reply firstFile.js:4]: I dont support hate speech you fucker

Comment (AlfredMadere) [reply firstFile.js:4]: Okay you right

Comment (MarcusTita) [reply firstFile.js:4]: But this variable is bad though

File: greatNewFeature.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
 //this might break uh oh 
-
+//nonononon
```
Comment (AlfredMadere) [inline greatNewFeature.js:2]: THis is a problematic comment it needs changing, it is not conventional

**Review by AlfredMadere — CHANGES_REQUESTED** [review]
There are a frew things to be changed, namely the nonono comment

**Pull request merged**